### ISSUE TITLE
"No data uploaded" message with custom input

### ISSUE THREAD

**Issue by silasmellor** [tbaccata/amica#7]
Hi, 
Firstly let me say i think this tool looks amazing. I am struggling a bit with the input formats however. I have an output from DIA-NN (report.pg-matrix.tsv), which contains proteinID, gene name and LFQ intensity for my runs. I have formatted this into the format required for custom file upload, including a specifications file to let amica know my headers. When i click analyze, i get a small popup in the right corner saying "no data uploaded". I am not sure what to make of this (amica reports upload complete when i add the proteingroups.txt and other files).
I would process with fragpipe or something else if that was easier, but to my knowledge fragpipe does not yet support DIApasef data.
Thanks in advance
Silas

**Comment by tbaccata** [comment]
Dear Silas,

does it work when all tab-separated files (design, contrasts, specification file) file extensions end with ".txt" (or .tsv, or .dat)?
I could reproduce the error message, when one of this files doesn't have an appropriate file extension.

The file extension for custom proteingroups file shouldn't matter.

Best,
Sebastian

**Comment by silasmellor** [comment]
All the files were saved from excel as .txt (tab separated). I tried both .txt and .tsv. I managed to get it to work by changing the column based on the example data. I only have proteinID, geneName and LFQ columns (as i understand it the DIA-NN output is already filtered to exclude one hit wonders so i left that out of the analysis). I am honestly not sure why the specifications file i originally supplied was not acceptable, but i suspect the issue is a mismatch somewhere in the columne headers and/or specification.
